.circleci/config.yml

```diff
@@ -75,45 +110,60 @@ workflows:
           dockerfile: sample/Dockerfile
           path: workspace
           extra-build-args: --compress
+          region: us-west-2
```

Comment:
I believe we do have this in our context. Rather than hard-coding it could we use `${AWS_REGION}`